src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 // write your code here
+const population = document.querySelectorAll('span.population');
+const totalElement = document.querySelector('span.total-population');
+const averageElement = document.querySelector('span.average-population');
+const populationNums = [];
+
+for (const populationElement of population) {
+  populationNums.push(+populationElement.innerText.split(',').join(''));
+}
+
+const total = populationNums.reduce((a, b) => a + b);
```

Comment:
use more meaningful variable names, in array methods too
It's hard to understand what is `a` and what is `b`